critiquebrainz/frontend/templates/search/results.html

```diff
@@ -101,5 +101,29 @@
       </td>
     </tr>
 
+  {% elif type=="recording" %}
+    <tr>
+      <td>
+        <a href="{{ url_for('recording.entity', id=result.id) }}">{{ result['title'] }}</a>
+      </td>
+      <td>
+        {% if result['length'] %}
+          {{ result['length'] | track_length_ms }}
+        {% else %}
+          -
+        {% endif %}
+      </td>
+      <td>
+        <a href="{{ url_for('artist.entity', mbid=result['artist-credit'][0].artist.id) }}">{{ result['artist-credit'][0]['artist']['name'] or '-' }}</a>
```

Comment:
This isn't strictly correct, it'd be nice to be able to render the full artist credit here. As we have to do this elsewehre, perhaps we should have a method that can do it for us

Reply:
Fixed.